packages/react-scripts/fixtures/kitchensink/integration/webpack.test.js

```diff
@@ -39,6 +38,7 @@ describe('Integration', () => {
       const doc = await initDOM('unknown-ext-inclusion')
 
       expect(doc.getElementById('feature-unknown-ext-inclusion').textContent).to.equal('Whoooo, spooky!.')
```

Comment:
Maybe it's worth making sure it's equal to `aFileWithExtUnknown`, same with the one above (the hard-coded text).